src/Shared/UnitTests/MockLogger.cs

```diff
@@ -302,47 +302,53 @@ internal void LoggerEventHandler(object sender, BuildEventArgs eventArgs)
         {
             if (eventArgs is BuildWarningEventArgs)
             {
-                BuildWarningEventArgs w = (BuildWarningEventArgs)eventArgs;
+                BuildWarningEventArgs w = (BuildWarningEventArgs) eventArgs;
 
                 // hack: disregard the MTA warning.
                 // need the second condition to pass on ploc builds
                 if (w.Code != "MSB4056" && !w.Message.Contains("MSB4056"))
                 {
-                    _fullLog.AppendFormat("{0}({1},{2}): {3} warning {4}: {5}\r\n",
+                    string logMessage = string.Format("{0}({1},{2}): {3} warning {4}: {5}",
```

Comment:
>logMessage [](start = 27, length = 10)

Would it be better to switch this and the other below to use string interpolation?